src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateSummary.jsx

```diff
@@ -79,7 +77,9 @@ class DateSummary extends React.Component {
   }
 }
 
-const DateGroupWrapper = styled(Flex)`
+const DateGroupWrapper = styled('div')`
```

Comment:
This was throwing react warnings re: `hasTime` being invalid DOM property (styling a native DOM element means emotion cleans up these props).